### PR TITLE
Add documentation for color codes to Notes tab

### PR DESCRIPTION
**Pull request by Wires77**
I'm not a huge fan of how this looks right now, so I'm open to suggestions on how to display this better.  Another option would be a group of buttons that would change the color.  More user-friendly, but also a lot more work.

image

### CODE REVIEW

File: src/Classes/NotesTab.lua [hunk 1]
```diff
@@ -13,12 +13,17 @@ local NotesTabClass = newClass("NotesTab", "ControlHost", "Control", function(se
 
 	self.lastContent = ""
 
-	self.controls.edit = new("EditControl", {"TOPLEFT",self,"TOPLEFT"}, 8, 8, 0, 0, "", nil, "^%C\t\n", nil, nil, 16)
+	local colorDesc = [[^7This field supports different colors.  Using the caret symbol (^) followed by a Hex code or a number will set the color.  Below are some common color codes PoB uses
```
Comment (ppoelzl) [inline src/Classes/NotesTab.lua:16]:
```suggestion
	local colorDesc = [[^7This field supports different colors.  Using the caret symbol (^) followed by a Hex code or a number will set the color.  Below are some common color codes PoB uses:
```

**Review by ppoelzl — APPROVED** [review]
LGTM as-is. We don't need to go overboard with editor features. If someone wants more control, there is more suitable doument creating software out there.

**Comment by zao** [comment]
Seems reasonable enough as long as we consider it a feature that user-sourced input is interpreted as invisible formatting and that the editing experience is going to be different when hitting backspace vs. moving the cursor around.
This shouldn't block any future work on Unicode support, which was my primary concern.

**Comment by Wires77** [comment]
Alright, I'm going to tweak the spacing and add a couple more colors, then merge it in.

**Review by ppoelzl — APPROVED** [review]
Yes, that should be more maintainable.

**Pull request merged**